### ISSUE TITLE
Board Restarts during Post Game SC Champions Check

### ISSUE THREAD

**Issue by sflems** [riffnshred/nhl-led-scoreboard#379]
**Describe the bug**
Board Restarts during Post Game SC Champions Check and gets stuck in restart loop.

**Logs**
```
2-06-15 23:03:40 DEBUG Starting new HTTP connection (1): statsapi.web.nhl.com:80
22-06-15 23:03:40 DEBUG http://statsapi.web.nhl.com:80 "GET /api/v1/schedule?date=2022-6-15&expand=schedule.linescore HTTP/1.1" 200 None
22-06-15 23:03:40 DEBUG Starting new HTTP connection (1): statsapi.web.nhl.com:80
22-06-15 23:03:40 DEBUG http://statsapi.web.nhl.com:80 "GET /api/v1/game/2021030411/feed/live?site=en_nhl HTTP/1.1" 200 None
22-06-15 23:03:41 DEBUG STREAM b'IHDR' 16 13
22-06-15 23:03:41 DEBUG STREAM b'IDAT' 41 3003
22-06-15 23:03:41 DEBUG STREAM b'IHDR' 16 13
22-06-15 23:03:41 DEBUG STREAM b'IDAT' 41 1630
22-06-15 23:03:41 DEBUG Starting new HTTP connection (1): statsapi.web.nhl.com:80
22-06-15 23:03:41 DEBUG http://statsapi.web.nhl.com:80 "GET /api/v1/game/2021030411/feed/live?site=en_nhl HTTP/1.1" 200 None
22-06-15 23:03:41 DEBUG STREAM b'IHDR' 16 13
22-06-15 23:03:41 DEBUG STREAM b'tEXt' 41 25
22-06-15 23:03:41 DEBUG STREAM b'iTXt' 78 883
22-06-15 23:03:41 DEBUG STREAM b'IDAT' 973 291
INFO: Validating config.json.....
INFO: config.json passes validation
22-06-15 23:03:58 INFO Scheduler started
22-06-15 23:03:58 DEBUG Looking for jobs to run
22-06-15 23:03:58 DEBUG No jobs; waiting until a job is added
22-06-15 23:03:58 INFO Added job "UpdateChecker.CheckForUpdate" to job store "default"
22-06-15 23:03:58 INFO Using cache directory: /root/.cache/lastversion.
22-06-15 23:03:58 INFO Created instance of GitHubRepoSession
22-06-15 23:03:58 DEBUG Looking for jobs to run
22-06-15 23:03:58 DEBUG Next wakeup is due at 2022-06-16 03:00:00-07:00 (in 14161.535157 seconds)
22-06-15 23:03:58 DEBUG Looking up "https://github.com/riffnshred/nhl-led-scoreboard/releases.atom" in the cache
22-06-15 23:03:58 DEBUG Current age based on date: 1254
22-06-15 23:03:58 DEBUG Freshness lifetime from max-age: 0
22-06-15 23:03:58 DEBUG Starting new HTTPS connection (1): github.com:443
22-06-15 23:03:58 DEBUG https://github.com:443 "GET /riffnshred/nhl-led-scoreboard/releases.atom HTTP/1.1" 304 0
22-06-15 23:03:58 INFO Got HTTP status code 200 from https://github.com/riffnshred/nhl-led-scoreboard/releases.atom
22-06-15 23:03:58 INFO Checking tag v1.6.9
22-06-15 23:03:58 INFO Sanitizing string v1.6.9 as a satisfying version.
22-06-15 23:03:58 INFO Parsed as Version OK
22-06-15 23:03:58 INFO String representation of version is 1.6.9.
22-06-15 23:03:58 DEBUG Looking up "https://api.github.com/repos/riffnshred/nhl-led-scoreboard/releases/tags/v1.6.9" in the cache
22-06-15 23:03:58 DEBUG Current age based on date: 1254
22-06-15 23:03:58 DEBUG Freshness lifetime from max-age: 60
22-06-15 23:03:58 DEBUG Starting new HTTPS connection (1): api.github.com:443
22-06-15 23:03:59 DEBUG https://api.github.com:443 "GET /repos/riffnshred/nhl-led-scoreboard/releases/tags/v1.6.9 HTTP/1.1" 304 0
22-06-15 23:03:59 INFO Got HTTP status code 200 from https://api.github.com/repos/riffnshred/nhl-led-scoreboard/releases/tags/v1.6.9
22-06-15 23:03:59 INFO Got formal release for tag v1.6.9
22-06-15 23:03:59 INFO Selected version as current selection: 1.6.9.
22-06-15 23:03:59 INFO Checking tag v1.6.8
22-06-15 23:03:59 INFO Sanitizing string v1.6.8 as a satisfying version.
22-06-15 23:03:59 INFO Parsed as Version OK
22-06-15 23:03:59 INFO String representation of version is 1.6.8.
22-06-15 23:03:59 INFO Tag v1.6.8 does not contain newer version than we already found
22-06-15 23:03:59 INFO Checking tag v1.6.7
22-06-15 23:03:59 INFO Sanitizing string v1.6.7 as a satisfying version.
22-06-15 23:03:59 INFO Parsed as Version OK
22-06-15 23:03:59 INFO String representation of version is 1.6.7.
22-06-15 23:03:59 INFO Tag v1.6.7 does not contain newer version than we already found
22-06-15 23:03:59 INFO Checking tag v1.6.6
22-06-15 23:03:59 INFO Sanitizing string v1.6.6 as a satisfying version.
22-06-15 23:03:59 INFO Parsed as Version OK
22-06-15 23:03:59 INFO String representation of version is 1.6.6.
22-06-15 23:03:59 INFO Tag v1.6.6 does not contain newer version than we already found
22-06-15 23:03:59 INFO Checking tag v1.6.5
22-06-15 23:03:59 INFO Sanitizing string v1.6.5 as a satisfying version.
22-06-15 23:03:59 INFO Parsed as Version OK
22-06-15 23:03:59 INFO String representation of version is 1.6.5.
22-06-15 23:03:59 INFO Tag v1.6.5 does not contain newer version than we already found
22-06-15 23:03:59 INFO Checking tag v1.6.4
22-06-15 23:03:59 INFO Sanitizing string v1.6.4 as a satisfying version.
22-06-15 23:03:59 INFO Parsed as Version OK
22-06-15 23:03:59 INFO String representation of version is 1.6.4.
22-06-15 23:03:59 INFO Tag v1.6.4 does not contain newer version than we already found
22-06-15 23:03:59 INFO Checking tag v1.6.3
22-06-15 23:03:59 INFO Sanitizing string v1.6.3 as a satisfying version.
22-06-15 23:03:59 INFO Parsed as Version OK
22-06-15 23:03:59 INFO String representation of version is 1.6.3.
22-06-15 23:03:59 INFO Tag v1.6.3 does not contain newer version than we already found
22-06-15 23:03:59 INFO Checking tag v1.6.2
22-06-15 23:03:59 INFO Sanitizing string v1.6.2 as a satisfying version.
22-06-15 23:03:59 INFO Parsed as Version OK
22-06-15 23:03:59 INFO String representation of version is 1.6.2.
22-06-15 23:03:59 INFO Tag v1.6.2 does not contain newer version than we already found
22-06-15 23:03:59 INFO Checking tag v1.6.1
22-06-15 23:03:59 INFO Sanitizing string v1.6.1 as a satisfying version.
22-06-15 23:03:59 INFO Parsed as Version OK
22-06-15 23:03:59 INFO String representation of version is 1.6.1.
22-06-15 23:03:59 INFO Tag v1.6.1 does not contain newer version than we already found
22-06-15 23:03:59 INFO Checking tag v1.6.0
22-06-15 23:03:59 INFO Sanitizing string v1.6.0 as a satisfying version.
22-06-15 23:03:59 INFO Parsed as Version OK
22-06-15 23:03:59 INFO String representation of version is 1.6.0.
22-06-15 23:03:59 INFO Tag v1.6.0 does not contain newer version than we already found
22-06-15 23:03:59 INFO Located the latest release at: https://github.com/riffnshred/nhl-led-scoreboard via feed mechanism
22-06-15 23:03:59 DEBUG Starting new HTTP connection (1): statsapi.web.nhl.com:80
22-06-15 23:03:59 DEBUG http://statsapi.web.nhl.com:80 "GET /api/v1/game/2021030411/feed/live?site=en_nhl HTTP/1.1" 200 None
22-06-15 23:03:59 DEBUG STREAM b'IHDR' 16 13
22-06-15 23:03:59 DEBUG STREAM b'IDAT' 41 3003
22-06-15 23:04:00 DEBUG STREAM b'IHDR' 16 13
22-06-15 23:04:00 DEBUG STREAM b'IDAT' 41 1630
Traceback (most recent call last):
  File "./src/main.py", line 156, in <module>
    run()
  File "./src/main.py", line 151, in run
    MainRenderer(matrix, data, sleepEvent).render()
  File "/home/pi/nhl-led-scoreboard/src/renderer/main.py", line 76, in render
    self.__render_game_day()
  File "/home/pi/nhl-led-scoreboard/src/renderer/main.py", line 183, in __render_game_day
    self.check_stanley_cup_champion()
AttributeError: 'MainRenderer' object has no attribute 'check_stanley_cup_champion'
```

**Your config file**
```
{
    "debug": true,
    "loglevel": "INFO",
    "live_mode": true,
    "preferences": {
        "time_format": "12h",
        "end_of_day": "12:00",
        "location": "Campbell River, BC",
        "live_game_refresh_rate": 10,
        "teams": [
            "Avalanche"
        ],
        "sog_display_frequency": 4,
        "goal_animations": {
            "pref_team_only": true
        }
    },
    "states": {
        "off_day": [
            "team_summary",
            "clock",
            "seriesticker"
        ],
        "scheduled": [
            "scoreticker",
            "seriesticker",
            "team_summary"
        ],
        "intermission": [
            "scoreticker",
            "seriesticker",
            "team_summary"
        ],
        "post_game": [
            "scoreticker",
            "clock",
            "stanley_cup_champions"
        ]
    },
    "boards": {
        "scoreticker": {
            "preferred_teams_only": false,
            "rotation_rate": 5
        },
        "seriesticker": {
            "preferred_teams_only": true,
            "rotation_rate": 5
        },
        "standings": {
            "preferred_standings_only": true,
            "standing_type": "division",
            "divisions": "atlantic",
            "conference": "eastern"
        },
        "clock": {
            "duration": 15,
            "hide_indicator": false,
            "preferred_team_colors": true,
            "clock_rgb": "",
            "date_rgb": "",
            "flash_seconds": false
        },
        "weather": {
            "enabled": false,
            "view": "full",
            "units": "metric",
            "duration": 60,
            "data_feed": "EC",
            "owm_apikey": "",
            "update_freq": 5,
            "show_on_clock": true,
            "forecast_enabled": false,
            "forecast_days": 3,
            "forecast_update": 1
        },
        "wxalert": {
            "alert_feed": "EC",
            "update_freq": 5,
            "nws_show_expire": false,
            "show_alerts": false,
            "alert_title": true,
            "scroll_alert": true,
            "alert_duration": 5,
            "show_on_clock": true
        }
    },
    "sbio": {
        "screensaver": {
            "enabled": false,
            "animations": true,
            "start": "22:00",
            "stop": "08:00",
            "data_updates": false,
            "motionsensor": false,
            "pin": 7,
            "delay": 30
        },
        "dimmer": {
            "enabled": false,
            "source": "software",
            "frequency": 5,
            "light_level_lux": 400,
            "mode": "always",
            "daytime": "",
            "nighttime": "",
            "offset": 0,
            "sunset_brightness": 5,
            "sunrise_brightness": 40
        },
        "pushbutton": {
            "enabled": false,
            "bonnet": false,
            "pin": 25,
            "reboot_duration": 2,
            "reboot_override_process": "",
            "display_reboot": true,
            "poweroff_duration": 10,
            "poweroff_override_process": "",
            "display_halt": true,
            "state_triggered1": "weather",
            "state_triggered1_process": ""
        }
    }
}
```

**Hardware you are using**
- Raspberry pi model - 
_4 (4GB)_

- The Matrix driver (Adafruit Bonnet or Adafruit HAT) - 
_BONNET_

- Anti-flicker mod or not? - 
_YES_

**Software**
- Which version of the software are you on?
_1.6.9_

**Comment by riffnshred** [comment]
Thanks !! Im on it !

**Comment by riffnshred** [comment]
this is fixed on 1.6.10
